install_gtk_workload.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# For some reason automatic workload manifest detection doesn't work (see https://github.com/GtkSharp/GtkSharp/issues/355#issuecomment-1446262239), so download and uzip mainfest file manually
+dotnet nuget add source --name nuget.org "https://api.nuget.org/v3/index.json"
+wget https://www.nuget.org/api/v2/package/gtksharp.net.sdk.gtk.manifest-$GtkSharpManifestVersion/$GtkSharpVersion -O gtksharp.net.sdk.gtk.manifest-$GtkSharpManifestVersion.nupkg            
+DOTNET_DIR=/home/runner/.dotnet
+WORKLOAD_MANIFEST_DIR=$DOTNET_DIR/sdk-manifests/$DotnetVersion/gtksharp.net.sdk.gtk
+mkdir -p $WORKLOAD_MANIFEST_DIR
+unzip -j gtksharp.net.sdk.gtk.manifest-$GtkSharpManifestVersion.nupkg "data/*" -d $WORKLOAD_MANIFEST_DIR/
+rm gtksharp.net.sdk.gtk.manifest-$GtkSharpManifestVersion.nupkg
+chmod 764 $WORKLOAD_MANIFEST_DIR/*
+dotnet workload search
+dotnet workload install gtk --skip-manifest-update
```

Comment:
@webwarrior-ws I was going to ask you to squash this change now with the commit that introduces this code, but I see now that there's no commit in this PR that introduces this code. I guess in the other PR I asked you to move this to a script because it looked like new code introduced in the PR, but if in this PR it looks like the code was there already, then please don't move it, as the move would be unrelated to the PR

Reply:
removed commit that moves the code in question